File: CarPoolSolution/Models/OfferRide.cs
using CarPool.Enums;

namespace CarPool.Models
{    
    public class OfferRide
    {
        public int Id { get; set; }        
        public int OfferId { get; set; }
        public Stop From { get; set; }
        public Stop To { get; set; }
        public string Date { get; set; }
        public string Time { get; set; }
        public int SeatsAvailable { get; set; }
        public int Price { get; set; }
        public int IssuerId { get; set; }
        public string Stops { get; set; }

    }
}

File: CarPoolSolution/CarPoolAngular/src/app/models/OfferRide.ts
import { Stops } from '../constants/Stops';

export class OfferRide {
  public offerId!: number;
  public From!: Stops | string;
  public To!: Stops | string;
  public Date!: string;
  public Time!: string;
  public seatsAvailable!: number;
  public price!: number;
  public Stops!: string | undefined;
  public IssuerId!:number;
  public id!: number;

  constructor(
    from: Stops | string,
    to: Stops | string,
    date: string,
    time: string,
    seatsAvailable: number,
    price: number,
    stops: string,
    issuerId: number
  ) {
    this.From = from;
    this.To = to;
    this.Date = date;
    this.Time = time;
    this.seatsAvailable = seatsAvailable;
    this.price = price;
    this.Stops = stops;
    this.IssuerId = issuerId;
  }
}
